### DISCUSSION TITLE
October Newsletter

### DISCUSSION THREAD

**Discussion by briancl2** [briancl2/CustomerNewsletter#7] [Customer Newsletter]
This is a personally curated newsletter for just my customers. The purpose is to pull together and share current resources, announcements, and events. I welcome your feedback, and please let me know if you would like to add or remove anyone from this list.

### Copilot
Before we get into the long list of Universe announcements, I want to share a useful [eBook](https://resources.github.com/artificial-intelligence/scaling-github-copilot-across-your-organization/) that GitHub just published. This eBook provides guidance for large enterprises rolling out Copilot at scale. Topics include how to build trust, tech readiness, training tips, how to encourage collaboration, and customer success stories. Highly recommended!

For the Universe launches, I broke out the highlights by their availability:
#### Generally Available
* [GitHub Copilot Metrics API GA](https://github.blog/changelog/2024-10-30-github-copilot-metrics-api-ga-release-now-available/) now available ([docs](https://docs.github.com/en/rest/copilot/copilot-metrics?apiVersion=2022-11-28))
* Improved Suggestions for [C#](https://github.blog/changelog/2024-10-29-improving-github-copilot-code-suggestions-in-visual-studio-for-c-developers/) and [C++](https://github.blog/changelog/2024-10-25-improving-github-copilot-c-completions-in-visual-studio/) in Visual Studio
* New in VS Code: [Copilot multi-file editing](https://github.blog/changelog/2024-10-29-multi-file-editing-code-review-custom-instructions-and-more-for-github-copilot-in-vs-code-october-release-v0-22/) ([docs](https://code.visualstudio.com/docs/copilot/copilot-edits))
#### Public Preview (access today with [pre-release terms](https://docs.github.com/en/site-policy/github-terms/github-pre-release-license-terms))
* Use [Copilot Workspace in the Pull Request](https://github.blog/changelog/2024-10-29-refine-and-validate-code-review-suggestions-with-copilot-workspace-public-preview/) to quickly refine, test, and iterate without having to leave your workflow. ([docs](https://docs.github.com/en/copilot/using-github-copilot/using-github-copilot-for-pull-requests/using-copilot-to-help-you-work-on-a-pull-request))
* When you’re done with your changes, [request an AI Code Review](https://github.blog/changelog/2024-10-29-github-copilot-code-review-in-github-com-public-preview/) from Copilot ([docs](https://docs.github.com/en/copilot/using-github-copilot/code-review/using-copilot-code-review))
* Users now have [Model Choice](https://github.blog/news-insights/product-news/bringing-developer-choice-to-copilot/) with Copilot. We are adding OpenAI o1-preview, o1-mini, Anthropic Claude 3.5 Sonnet, and Google Gemini 1.5 Pro to a new model selector for a growing list of Copilot features ([docs](https://docs.github.com/en/copilot/using-github-copilot/using-claude-sonnet-in-github-copilot))
* Copilot Extensions: Chat with Azure API Docs or troubleshoot apps directly in Copilot Chat with the [@azure extension](https://techcommunity.microsoft.com/t5/microsoft-developer-community/streamline-your-azure-workflow-introducing-github-copilot-for/ba-p/4276989)
* Copilot code completions in [Apple Xcode](https://github.blog/changelog/2024-10-29-github-copilot-code-completion-in-xcode-is-now-available-in-public-preview/)
#### Waitlists
* Join the [GitHub Copilot Upgrade Assistant for Java](https://github.com/features/preview/copilot-java-upgrade) waitlist
* Build entire applications in natural language using [GitHub Spark](http://gh.io/spark-signup) ([video](https://www.youtube.com/watch?v=oM2amcnVmzM))

### GHAS
* Leverage the power of Copilot Autofix to triage and remediate thousands of vulnerabilities at a time with [Security Campaigns](https://github.blog/changelog/2024-10-29-security-campaigns-with-copilot-autofix-are-now-in-public-preview/) ([docs](https://docs.github.com/en/enterprise-cloud@latest/code-security/securing-your-organization/fixing-security-alerts-at-scale/about-security-campaigns))
* Extend Copilot Autofix to [Dependabot](https://github.com/features/preview/copilot-autofix-for-dependabot) to suggest fixes for updating dependencies at scale. 
* Copilot Autofix now [supports third party scanning tools](https://github.blog/changelog/2024-10-29-copilot-autofix-now-supports-partner-code-scanning-tools) in addition to GHAS

### Upcoming Webinars
* Nov 5 - [GitHub Copilot plans and features](https://developer.microsoft.com/en-us/reactor/events/23718/)
* Nov 6 - [(EMEA) Intro to GitHub Copilot](https://resources.github.com/intro-to-copilot-november6th/)
* Nov 7 - [Deploying with GitHub @azure and @workspace Agents](https://developer.microsoft.com/en-us/reactor/events/23814/)
* Nov 7 - [Copilot Extensions with Swimm, Crowdbotics, Pinecone, & Faros](https://resources.github.com/events/webinar-mastering-github-with-copilot-nov7/)
* Nov 12 - [Developer Use Cases for AI with GitHub Copilot](https://developer.microsoft.com/en-us/reactor/events/23719/)
* Nov 18 - [KPMG & GitHub - Autofix with Copilot to Remediate at Scale](https://developer.microsoft.com/en-us/reactor/events/23974/)
* Nov 19 - [Testing with GitHub Copilot](https://developer.microsoft.com/en-us/reactor/events/23722/)
* Nov 20 - [GitHub Q4 Roadmap Webinar](https://resources.github.com/webcasts/github-roadmap-webinar-q4-americas/)
* Nov 20 - [(EMEA) Intro to GitHub Copilot](https://resources.github.com/intro-to-copilot-november20th/)
* Dec 4 - [(EMEA) Intro to GitHub Copilot](https://resources.github.com/intro-to-copilot-december4th/)
* Dec 12 - [Prompt Engineering 300](https://developer.microsoft.com/en-us/reactor/events/23687/)

### Upcoming Events
* Las Vegas - December 2-6 - [GitHub at AWS re:Invent](https://resources.github.com/events/aws-reinvent24/)

**Comment by lenznerm** [comment]
@briancl2 - if of interest, this was just sent out - https://github.registration.goldcast.io/series/ff0939de-f20a-4395-80ff-4bc606e356fd

Copilot Fridays... ;-)
